ncbi-amrfinderplus/3.10.24/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:jammy as app
+
+# For easy upgrade later. ARG variables only persist during build time
+ARG AMRFINDER_VER="3.10.24"
+
+# now installed via APT; have to use ubuntu:jammy to get these later versions
+#ARG BLAST_VER="2.12.0"
+#ARG HMMER_VER="3.3.2"
+
+# to prevent tzdata from asking for a region during apt updates; ARG so that variable only 
+# persists at buildtime
+ARG DEBIAN_FRONTEND=noninteractive
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="NCBI AMRFinderPlus"
+LABEL software.version="${AMRFINDER_VER}"
+LABEL description="NCBI resistance gene detection tool"
+LABEL website="https://github.com/ncbi/amr"
+LABEL maintainer1="Kelsey Florek"
+LABEL maintainer1.email="kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Anders Goncalves da Silva"
+LABEL maintainer3.email="andersgs@gmail.com"
+
+# install dependencies, clean up apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ wget \ 
+ curl \
+ ca-certificates \
+ ncbi-blast+ \
+ hmmer && \
+ apt-get autoclean && \
+ rm -rf /var/lib/apt/lists/*
+
+# ncbi-blast+ installed via apt is v2.12.0
```

Comment:
I don't think we need these comments twice so close together. I don't think you need to remove them, it's just a personal preference.